packages/react-composites/src/composites/ChatComposite/file-sharing/FileCardGroup.tsx

```diff
@@ -19,11 +19,10 @@ const fileCardGroupClassName = mergeStyles({
   flexFlow: 'row wrap',
   '& > *': {
     margin: '0.5rem',
-    marginLeft: '0 rem',
     marginRight: 'auto'
   },
-  ':not(:first-child)': {
-    marginLeft: '0.5rem'
+  '& > *:not(:first-child)': {
```

Comment:
Can you add a comment why this specific selector is needed? Raw css like this is very prone to regressions so any info really helps if someone needs to make changes to this further down the line (i.e. the comment should give a developer confidence that if they change css in this class they do not cause a regression)